src/patient-registration/section/relationship-section.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react';
+import styles from './section.css';
+import { RelationshipInput } from '../input/custom-input/relationship-input.component ';
+
+interface RelationshipProps {
+  setFieldValue(field: string, value: any, shouldValidate?: boolean): void;
```

Comment:
Is there a better typing for `value: any`?

Reply:
My best guess is that we would have to create a `type` having the different form values....[also see here](https://formik.org/docs/api/formik#setfieldvalue-field-string-value-any-shouldvalidate-boolean--void)

Reply:
Is this related to your comment above regarding `value: any`?

Reply:
yes